docker-storage-setup.sh

```diff
@@ -1191,7 +1194,10 @@ if [ $# -eq 2 ]; then
     STORAGE_IN_FILE=$1
     STORAGE_OUT_FILE=$2
 else
-    if [ $# -gt 0 ]; then
+    if [ $# -eq 0 ]; then
+	# Default for the docker use case
+	CONTAINER_THINPOOL="docker-pool"
```

Comment:
What happens if docker user specifies CONTAINER_THINPOOL. I think we should still honor that?

So this probably should be 

` [  - z "$CONTAINER_THINPOOL ] && CONTAINER_THINPOOL="docker-pool" ?